File: image-train-filters-app/src/main/java/org/jesperancinha/itf/android/common/ChartizateSurfaceView.java
package org.jesperancinha.itf.android.common;

import android.content.Context;
import android.util.AttributeSet;
import android.view.SurfaceView;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class ChartizateSurfaceView extends SurfaceView {
    private int color;

    public ChartizateSurfaceView(Context context, AttributeSet attrs) {
        super(context, attrs);
    }
}

File: app-g2/app/src/main/java/org/jesperancinha/itf/app/common/ChartizateSurfaceView.kt
package org.jesperancinha.itf.app.common

import android.content.Context
import android.util.AttributeSet
import android.view.SurfaceView

data class ChartizateSurfaceView(val context: Context, val attrs: AttributeSet) :
    SurfaceView(context, attrs) {
    private val color = 0
}